package/native-package/package.json

```diff
@@ -40,9 +40,7 @@
     "react-native-haptic-feedback": "^1.14.0",
     "react-native-image-crop-picker": "^0.38.0",
     "react-native-image-resizer": ">=1.4.2",
-    "react-native-share": ">=4.1.0"
-  },
-  "optionalDependencies": {
```

Comment:
`@react-native-clipboard/clipboard` under `optionalDependencies` caused a false positive detection when running `require("@react-native-clipboard/clipboard")`

Reply:
This has resulted in a crash

Reply:
This is not the right way.. the right way for using optional dependencies is like this

```
{
  "peerDependencies": {
    "@react-native-clipboard/clipboard": "^1.11.1"
  },
  "peerDependenciesMeta": {
    "@react-native-clipboard/clipboard": {
      "optional": true
    }
  }
}
```

ref: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#peerdependenciesmeta